src/azure-cli/azure/cli/command_modules/backup/custom_help.py

```diff
@@ -108,9 +109,17 @@ def validate_policy(policy):
     validate_object(policy, "Policy not found. Please provide a valid policy_name.")
 
 
+def validate_protectable_item(protectable_item):
+    validate_object(protectable_item, "Protectable item not found. Please provide a valid protectable_item_name.")
+
+
+def validate_azurefileshare_item(azurefileshare_item):
+    validate_object(azurefileshare_item, "Azure File Share item not found. Please provide a valid azure_file_share.")
+
+
 def validate_object(obj, error_message):
     if obj is None:
-        raise ValueError(error_message)
+        raise ResourceNotFoundError(error_message)
```

Comment:
Is `obj` an Azure resource? If not, maybe we should use `ValidationError` here.

Reply:
Yes. obj can be a container, policy, protectable item, azure file share, etc.